src/extension/iSamples_listFacet.js

```diff
@@ -193,12 +108,7 @@ class ListFacet extends React.Component {
             {label}
           </h5>
         </header>
-
-        {expanded ?
-          (customList ?
-            <CustomizedTreeView label={label} value={value} facetCounts={facetCounts} facetValues={facetValues} onClick={this.handleClick} hierarchy= {this.hierarchy}/>
-            : prevNode)
-          : null}
+        {expanded ? prevNode : null}
```

Comment:
Remove the code portion of rendering a custom list(i.e., hierarchy facet) as we separated this out into `iSamples_hierarchyFacet`